src/coreclr/ToolBox/superpmi/mcs/verbildump.cpp

```diff
@@ -90,7 +90,12 @@ void DumpSigToConsoleBare(MethodContext* mc, CORINFO_SIG_INFO* pSig)
     for (int i = 0; i < (int)pSig->numArgs; i++)
     {
         DWORDLONG          dl;
+#if defined(TARGET_LOONGARCH64)
+        int                  flags;
+        CorInfoTypeWithMod type = mc->repGetArgType(pSig, currentItem, (CORINFO_CLASS_HANDLE*)&dl, &flags, &exceptionCode);
```

Comment:
This and number and number of other changes were for getArgType2. They are not necessary anymore.

Reply:
OK, Thanks for your review.
I will delelte them.